src/features/projects/components/maps/Markers.tsx

```diff
@@ -32,7 +33,13 @@ export default function Markers({
   const handleClose = () => {
     setOpen(false);
   };
-
+  const markerBackgroundColor = (project) => {
+    return project.isTopProject
```

Comment:
How about using `project.isTopProject && project.isApproved` as at https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1794 or is `project.isApproved` for this coloring use case?